dependency/engine_test.go

```diff
@@ -166,13 +167,13 @@ func (s *EngineSuite) TestStartGetUndeclaredName(c *gc.C) {
 		// Install another task with an undeclared dependency on the started task.
 		done := make(chan struct{})
 		err = engine.Install("other-task", dependency.Manifold{
-			Start: func(context dependency.Context) (worker.Worker, error) {
-				err := context.Get("some-task", nil)
+			Start: func(ctx context.Context, container dependency.Container) (worker.Worker, error) {
```

Comment:
Why call it a 'snapshot' elsewhere but a 'Container' here.
It seems confusing, and Container often means the LXD/Docker/K8s thing.